src/kudu/consensus/consensus_peers.cc

```diff
@@ -307,12 +309,8 @@ void Peer::SendNextRequest(bool even_if_queue_empty) {
   // that this object outlives the RPC.
   shared_ptr<Peer> s_this = shared_from_this();
 
-  // TODO(mpercy): An error here means that the peer is not in the config. Need
-  // to see whether we can remove this CHECK, since in that case, what is the
-  // state of the current Peer object?
-  string next_hop_uuid;
-  CHECK_OK(queue_->GetNextRoutingHopFromLeader(peer_pb().permanent_uuid(), &next_hop_uuid));
-
+  // TODO: Refactor this code. Ideally all fields in 'request_' related to
+  // proxying should be set inside PeerMessageQueue::RequestForPeer()
```

Comment:
is this comment here for the set_proxy_hops_remaining variable?

Reply:
Yes, will make it explicit